### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by JustinStitt**
1) Add CI matrix to dropdown 
	- This also forces a scrollbar at the bottom of the matrix making for easier horizontal scrolling (at least on Chrome).

2) Cross out Meetups that have ~already occurred~.

3) Add new heading for architecture support

4) Change Hangouts to Google Meet as [it is now dead ☠](https://arstechnica.com/gadgets/2022/06/google-hangouts-finally-gets-a-shutdown-date-november-2022/)

### CODE REVIEW

**Comment by JustinStitt** [comment]
You can check the result here (on my fork): https://github.com/JustinStitt/ClangBuiltLinux.github.io

**Comment by kees** [comment]
I really like the drop-down. I find the line-through to be weird to read. I think it'd be better just to have "Upcoming Meetups" and "Past Meetups" maybe?

**Comment by JustinStitt** [comment]
> I really like the drop-down. I find the line-through to be weird to read. I think it'd be better just to have "Upcoming Meetups" and "Past Meetups" maybe?

I thought of doing past/present meetups but I felt as though I'd need a comprehensive list of _all_ past meetups to justify that. I don't want to just put two of the N number of past meetups as that would be misleading as to how many meetups have occurred.

Does anyone have the full list of all previous meetups (is it just these two?)

**Comment by nickdesaulniers** [comment]
> Cross out Meetups that have already occurred.

@metti just fixed these in https://github.com/ClangBuiltLinux/ClangBuiltLinux.github.io/pull/59

File: readme.md [hunk 1]
```diff
@@ -47,7 +53,9 @@
 - Discord: `#clang-built-linux` in the [LLVM server](https://discord.gg/xS7Z362)
 - Bi-weekly video meeting
   - [Calendar](https://calendar.google.com/calendar/embed?src=google.com_bbf8m6m4n8nq5p2bfjpele0n5s%40group.calendar.google.com)
-  - [Hangouts Meet](https://meet.google.com/yjf-jyqk-iaz)
+  - [Google Meet](https://meet.google.com/yjf-jyqk-iaz)
```
Comment (nickdesaulniers) [inline readme.md:56]: send as separate PR

File: readme.md [hunk 2]
```diff
@@ -47,7 +53,9 @@
...
+
+## Architecture Support
```
Comment (nickdesaulniers) [inline readme.md:58]: send as separate PR

**Review by nickdesaulniers — COMMENTED** [review]
We're still going to have the loading issue for tiles.

Also, on your fork, I still have the same issues grabbing the horizontal scroll bar. Maybe depends on the host os? (I'm on CrOS).

**Comment by JustinStitt** [comment]
Closing this PR and instead opening https://github.com/ClangBuiltLinux/ClangBuiltLinux.github.io/pull/63 to handle the readme improvements (sans dropdown menu). The dropdown menu will come in a different PR as well.
